mozilla-b2g/gaia
# [settings] icon cleanup

- use the latest Settings icons, see #5206 and #5207
- get the Wi-Fi signal level icons back (fixes regression of #5267)
- align Bluetooth device icons properly, see #5261
- remove unused files
- ensure all images follow the `whatever_file_name.png` guideline because everyone looooves underscores
